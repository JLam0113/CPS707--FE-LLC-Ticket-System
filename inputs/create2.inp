login
user1
create
user2
FS
exit
