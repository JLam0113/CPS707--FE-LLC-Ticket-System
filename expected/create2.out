Welcome, please enter a command.
Please enter your username
Privileged Login successful, now logged in as user1, please enter a command
Create account selected, please enter the username:
Please enter the account type:
You are not authorized to do that. please enter a command
Session ended
